### PR TITLE
Bug 1991357: bump library.go

### PR DESCRIPTION
**Pull request by aojea**
Fix the bug on the installer_controller  to only be degraded if lastFailedRevision = TargetRevision

### CODE REVIEW

File: bindata/assets/alerts/api-usage.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
               a successful upgrade to the next cluster version.
               Refer to `oc get apirequestcounts {{ $labels.resource }}.{{ $labels.version }}.{{ $labels.group }} -o yaml` to identify the workload.
           expr: |
-            group(apiserver_requested_deprecated_apis{removed_release="1.22"}) by (group,version,resource) and (sum by(group,version,resource) (rate(apiserver_request_total{system_client!="kube-controller-manager",system_client!="cluster-policy-controller"}[4h]))) > 0
+            group(apiserver_requested_deprecated_apis{removed_release="1.23"}) by (group,version,resource) and (sum by(group,version,resource) (rate(apiserver_request_total{system_client!="kube-controller-manager",system_client!="cluster-policy-controller"}[4h]))) > 0
```
Comment (aojea) [inline bindata/assets/alerts/api-usage.yaml:18]: @sanchezl is this the right fix? can you please take a look?

**Comment by aojea** [comment]
The test `TestAPIRemovedInNextReleaseInUse` is passing now
/retest
/assign @sanchezl

File: bindata/assets/alerts/api-usage.yaml [hunk 2]
```diff
@@ -27,7 +27,7 @@ spec:
               a successful upgrade to the next EUS cluster version.
               Refer to `oc get apirequestcounts {{ $labels.resource }}.{{ $labels.version }}.{{ $labels.group }} -o yaml` to identify the workload.
           expr: |
-            group(apiserver_requested_deprecated_apis{removed_release=~"1\\.2[123]"}) by (group,version,resource) and (sum by(group,version,resource) (rate(apiserver_request_total{system_client!="kube-controller-manager",system_client!="cluster-policy-controller"}[4h]))) > 0
+            group(apiserver_requested_deprecated_apis{removed_release=~"1\\.2[1234]"}) by (group,version,resource) and (sum by(group,version,resource) (rate(apiserver_request_total{system_client!="kube-controller-manager",system_client!="cluster-policy-controller"}[4h]))) > 0
```
Comment (sanchezl) [inline bindata/assets/alerts/api-usage.yaml:30]: No need to bump this here. Please revert.

**Pull request merged**